hivemind_etl/activities.py

```
@@ -1,91 +1,20 @@
 import logging
 from typing import Any
 
-from temporalio import activity, workflow
+from hivemind_etl.website.activities import (
+    get_hivemind_website_comminities,
```

Ruff (0.8.2)

`hivemind_etl.website.activities.get_hivemind_website_comminities` imported but unused

Remove unused import

(F401)